default/AI/AIDependencies.py

```diff
@@ -26,6 +26,8 @@
                   fo.planetSize.gasGiant: -1
                   }
 
+SUPPLY_MOD_SPECIALS = {'WORLDTREE_SPECIAL':{-1:1}}
```

Comment:
I'm not understanding your comment -- the -1 size is used to designate a size-independent contribution.  Making a function and adding some more explanatory comments sounds like a fine idea.